easybuild/easyconfigs/g/gffcompare/gffcompare-0.10.6-GCCcore-7.3.0.eb

```
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'gffcompare'
+version = '0.10.6'
+
+homepage = 'https://github.com/gpertea/%(namelower)s'
+description = """gffcompare provides classification and reference annotation mapping and
```

With capital letters like in the existing easyconfig:

```suggestion
description = """GffCompare provides classification and reference annotation mapping and
```